Makefile.build

```diff
@@ -11,6 +11,9 @@
 # This makefile is normally called by the main Makefile after setting
 # some variables.
 
+# Cleanup environment (avoids filling it up)
+unexport $(COQ_EXPORTED)
```

Comment:
Looks like it worked.

Reply:
> Looks like it worked.

Cool! So indeed this provides a path to enable native on Windows I think.

As for Dune, let's see what we do.